phpstan.neon

```diff
@@ -1,6 +1,5 @@
 parameters:
 	ignoreErrors:
-		- '#Parameter \#1 \$obj of function spl_object_hash expects object, callable given.#'
```

Comment:
The parameter name got changed from `$obj` to `$object` and now we cannot have one rule to match both variants. Ignoring the entire line instead now.

Reply:
Works for me 👍